src/components/KuadrantOverviewPage.tsx

```diff
@@ -285,6 +601,107 @@ const KuadrantOverviewPage: React.FC = () => {
+                          direction={{ default: 'column' }}
+                          alignItems={{ default: 'alignItemsCenter' }}
+                        >
+                          <strong style={{ fontSize: '1.3rem' }}>{gateways.length}</strong>
+                          <span>Total Gateways</span>
+                        </Flex>
+                      </FlexItem>
+
+                      {/* Healthy Gateways */}
+                      <FlexItem>
+                        <Flex
+                          direction={{ default: 'column' }}
+                          alignItems={{ default: 'alignItemsCenter' }}
+                        >
+                          <strong style={{ fontSize: '1.3rem' }}>
+                            <GreenCheckCircleIcon size="md" />{' '}
+                            <span style={{ margin: '5px' }}>{healthyCount}</span>
+                          </strong>
+                          <Tooltip
+                            content={
+                              <div>
+                                {t(
+                                  'A healthy gateway has a `true` status for the `Accepted` and `Programmed` conditions.',
+                                )}
+                              </div>
+                            }
+                          >
+                            <span>Healthy Gateways</span>
+                          </Tooltip>
+                        </Flex>
+                      </FlexItem>
+
+                      {/* Unhealthy Gateways */}
+                      <FlexItem>
+                        <Flex
+                          direction={{ default: 'column' }}
+                          alignItems={{ default: 'alignItemsCenter' }}
+                        >
+                          <strong style={{ fontSize: '1.3rem' }}>
+                            <YellowExclamationTriangleIcon size="md" />{' '}
+                            <span style={{ margin: '5px' }}>{unhealthyCount}</span>
+                          </strong>
+                          <Tooltip
+                            content={
+                              <div>
+                                {t(
+                                  'An unhealthy gateway has a `false` status for the `Accepted` and/or `Programmed` conditions.',
+                                )}
+                              </div>
+                            }
+                          >
+                            <span>Unhealthy Gateways</span>
+                          </Tooltip>
+                        </Flex>
+                      </FlexItem>
+                    </Flex>
+                  </CardBody>
+                </CardTitle>
+              </Card>
+            </FlexItem>
+          </Flex>
+
+          <Flex className="pf-u-mt-xl">
+            <FlexItem flex={{ default: 'flex_1' }}>
+              <Card>
+                <CardTitle>
+                  <Title headingLevel="h2">{t('Gateways - Traffic Distribution')}</Title>
```

Comment:
I wonder if we're missing a trick by not merging the `Gateways` & `Gateways - Traffic Distribution` cards together into just `Gateways`.

Crudely, something like (with the create gateway button inline with the resource list):
![Screenshot 2025-03-21 at 12 02 02](https://github.com/user-attachments/assets/f7bf61dc-651a-4361-9c30-f25004452178)

Reply:
I had a chat with Joey about this.
The first bar is meant as a 'Quick Insight' for the PE to gain a comprehensive understanding of gateways.
There are some other potential things that may land in there, like alerts.
Therefore, it might not make sense, or be scalable to have the bar combined with the gateway list.